nuxeo-runtime/nuxeo-launcher-commons/src/test/resources/configurator/nuxeo.conf

```diff
@@ -21,3 +21,12 @@ nuxeo.wizard.done=false
 test.sampled.prop2=someValue
 
 nuxeo.force.generation=true
+
+nuxeo.db.host=${env:NUXEO_DB_HOST}
+
+org.nuxeo.fake.vindoz=${env?:NUXEO_UNKNOWN_VAR}
```

Comment:
`org.nuxeo.fake.vindoz` is about ConfigurationGenerator.PARAM_FAKE_WINDOWS in the test

the test case for true is on `nuxeo.env.prop4`